libs/ebpfnetsh/programs.cpp

```diff
@@ -492,22 +494,22 @@ handle_ebpf_show_programs(
         return status;
     }
 
-    // If the user specified a filename and no level, default to verbose.
-    if (tags[3].bPresent && !tags[2].bPresent) {
+    // If the user specified an ID and no level, default to verbose.
+    if (tags[6].bPresent && !tags[2].bPresent) {
```

Comment:
Can we use an enum here instead of numeric constants?

Reply:
Agree would probably be good (though the code never did before and the other netsh helpers don't either)

Reply:
Doing this in my next changlist along with other netsh changes beyond this PR.

Reply:
This is in PR #553